openshift/ci-operator/generate-ci-config.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-branch=${1-'knative-v0.6.0'}
+branch=${1-'knative-v0.5.0'}
```

Comment:
Should be 0.6 ? We're on release-0.6 branch

Reply:
:-)